docker/launchTomcat.sh

```diff
@@ -1,14 +1,24 @@
 #!/bin/bash
 
 export JPDA_ADDRESS="*:5050"
+
 VISUALIZE_FITS_SEARCH_PATH=${VISUALIZE_FITS_SEARCH_PATH:-''}
 START_MODE=${START_MODE:-run}
 NAME=${BUILD_TIME_NAME:-"ipac/firefly"}
 
 ADMIN_USER=${ADMIN_USER:-admin}
-ADMIN_PASSWORD=${ADMIN_PASSWORD:-`echo $RANDOM | base64 | head -c 8`}
+if [[ -z "${ADMIN_PASSWORD}" ]]; then
+  ADMIN_PASSWORD=$(echo "${RANDOM}" | base64 || true)
+  ADMIN_PASSWORD=$(echo "${ADMIN_PASSWORD}" | head -c 8)
+fi
 USE_ADMIN_AUTH=${USE_ADMIN_AUTH:-"true"}
+EXTENSIONS_DIR=${EXTENSIONS_DIR:-${CATALINA_HOME}/extensions}
 
+# Check if CATALINA_HOME is set, if not, exit the script
+if [[ -z "${CATALINA_HOME}" ]]; then
+  echo "CATALINA_HOME enviroment variable is not set. Exiting..."
+  exit 1
+fi
```

Comment:
I don't believe this is necessary, as the base image we use should always have it set. If it's not set, we should configure it to the correct location rather than allowing it to fail.

Reply:
Sounds good.